src/binary.ts

```diff
@@ -312,14 +222,11 @@ export class Binary {
   }
 
   /** @internal */
-  static fromExtendedJSON(
-    doc: { $type: string; $binary: string | { subType: string; base64: string } },
-    options?: EJSONOptions
-  ): Binary {
+  static fromExtendedJSON(doc: BinaryEJSON, options?: EJSONOptions): Binary {
```

Comment:
nit: should our "extended JSON" types just be suffixed/prefixed with `Extended`? Like `BinaryExtended`, `ExtendedUInt32`, etc? Just a thought.

Reply:
We have "NODE-2723 Add type information for EJSON" so we can figure that out there.
I do think `BinaryExtended` or `ExtendedBinary` could be confusing _maybe? maybe not_ because if your in the domain of EJSON then its obvious, but it does seem to imply its an extension of the Binary class 🤔 def could come up with something better than tacking on `EJSON`